jenkins/JenkinsFile-rebuild-docker-images

```diff
@@ -300,6 +300,15 @@ pipeline {
   } // stages
   post {
     success {
+
+      // When the build is successful, trigger a validation build, using the tag just produced.
+      build job: 'daily-docker-image-validate',
```

Comment:
should any of these go in try/finally block?

Reply:
You mean a try/catch to check whether the `build` call here works? Not sure we need it, if that is the case. I it will only fail in case the job doesn't exist or there is a critical internal issue on Jenkins. In both cases we want to know and hence the job will fail.

Reply:
Well I'm thinking it may be best to not fail the job in case of a discord server side issue is all

Reply:
so originally my thought was coming from: which of these steps is more likely to fail but in a way which we want the build to continue? then i thought best was to put the more-likely-to-fail thing in a try block. but looking at this further, the thing that may fail but we want build to continue is the success notification; so given it's placed second it's actually fine without a try/finally here.